src/pal/tests/palsuite/c_runtime/_finitef/test1/test1.c

```diff
@@ -37,7 +37,7 @@ If E=0 and F is zero and S is 0, then V=0
 
 #define TO_FLOAT(x)    (*((float*)((void*)&x)))
 
-int __cdecl main(int argc, char **argv)
+int __cdecl main(int argc, const char **argv)
```

Comment:
if (PAL_Initialize(argc, argv) != 0)
                              ^
src/pal/tests/palsuite/common/palsuite.h:21:0,
    from src/pal/tests/palsuite/c_runtime/asinhf/test1/test1.c:18:
src/pal/inc/pal.h:374:1: note: expected ‘const char * const*’ but argument is of type ‘char **’

I attached the makefile hack to here: https://gist.github.com/franksinankaya/1b4e5e9d1cf9cfc28af27bc2fe03f0e5#file-coreclr-gnu-compile-for-pal

once you do this, you can compile with

./build.sh -gcc

if you want to see the warnings yourself using the gnuport_pal_2 branch here:
https://github.com/franksinankaya/coreclr.git

Reply:
I see, I've googled a bit and found it is a known inability of C. But casting from `char**` to `char * const *` is fine and that would be completely sufficient for the PAL_Initialize argv argument. Could you please change the PAL_Initialize argv argument to `char * const argv[]` instead of the change in the tests? I've tried and it seems to be ok (you'll just need fix one pal test - remove a cast in there).

Reply:
done